reference/docs-conceptual/install/Installing-PowerShell-Core-on-macOS.md

```diff
@@ -14,7 +14,7 @@ PowerShell 7.0 or higher require macOS 10.13 and higher. All packages are availa
 >
 > The `/usr/local/microsoft/powershell/6` folder is replaced by `/usr/local/microsoft/powershell/7`.
 >
-> If you need to run and older version of PowerShell core side-by-side with PowerShell 7.1, install
+> If you need to run an older version of PowerShell core side-by-side with PowerShell 7.1, then simply install
```

Comment:
```suggestion
> If you need to run an older version of PowerShell core side-by-side with PowerShell 7.1, install
```

Thanks for catching the typo. 

As for  "then simply": The current structure is correct and for the most part, we try and avoid using filler words where where don't need them.